libs/feature/record/src/lib/record-metadata/record-metadata.component.html

```diff
@@ -26,6 +26,27 @@
         </gn-ui-metadata-info>
       </div>
       <div>
+        <div
+          *ngIf="lightbox$ | async as lightbox"
+          data-cy="record-thumbnail"
+          class="flex-shrink-0 bg-gray-100 rounded-lg overflow-hidden w-full border border-gray-300 h-36 group-hover:shadow-xl group-hover:border-0 mb-3"
+        >
+          <gn-ui-thumbnail
+            class="relative h-full w-full"
+            [thumbnailUrl]="(facade.metadata$ | async).thumbnailUrl"
+            fit="cover"
+          ></gn-ui-thumbnail>
+          <div class="relative">
+            <gn-ui-button
+              class="absolute bottom-2 right-2"
+              [type]="'outline'"
```

Comment:
probably wrong type compare to the mockup (should be secondary bg).

Reply:
https://geonetwork.github.io/geonetwork-ui/main/storybook/demo/?path=/story/inputs-buttoncomponent--primary&args=type:secondary

Out of our 'default' buttons. In this case, is it better to follow the mockup or to implement one of our button ?

Reply:
You are right, and I think we need more convention about button styling. We should not have so many different button types...
Anyway, I think that `gnui-btn` should be replaced by a TW component, keep it like this we'll talk about that later, thanks.

Reply:
we can simply add a new button type, like `secondary-outline`. I think it was the right choice to use the `gn-ui-button` component, our strategy is to rely as much as possible on reusable low-level presentation components to avoid having too many TW classes in the HTML.

Reply:
I do prefer

`<button class="btn-secondary">` over `<gnui-button type="secondary">`,  which just manages CSS.
It might be a matter of taste, but TW component is
- no code
- no maintenance
- easily overridden in the app

Reply:
It could be great if we switch to scss, it could allow to write our buttons this way (with tailwind classes) for example: 
```scss
.btn {
  @apply px-2 py-1;
  .btn-secondary {
    @apply bg-secondary;
  }
  ...
}
```

```html 
<button class="btn btn-secondary" ...
```

Reply:
As far as I know, you can't for `@apply`. 
And for sure, you cannot do nested classes, which is quite useful.

Reply:
This is what I tested in `button.component.css`

```css
.btn-text-white {
  @apply text-white hover:text-primary-lighter;
}
.btn-text-white-a {
  @apply btn-text-white;
}
```

Reply:
Strange, we must have something else which preprocess our css ? 
https://caniuse.com/sr_css-apply

Reply:
https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply